include/aspect/boundary_temperature/dynamic_core.h

```diff
@@ -109,8 +109,8 @@ namespace aspect
          * at the inner and outer boundaries.
          *
          * @param boundary_indicator The boundary indicator of the part of the boundary
-         *   of the domain on which the point is located at which we are requesting the
```

Comment:
I normally go through the diffs manually and only apply it in wrong places but not in locations like here where the indentation makes sense. Thoughts?